pkg/render/kube-controllers.go

```diff
@@ -197,6 +197,18 @@ func (c *kubeControllersComponent) controllersRole() *rbacv1.ClusterRole {
 				Resources: []string{"clusterroles", "clusterrolebindings"},
 				Verbs:     []string{"watch", "list", "get"},
 			},
+			{
+				APIGroups:     []string{""},
```

Comment:
After chatting about this with @doublek we may need to make sure that we're restricting access appropriate, so that kube controllers can only access these resources in the tigera-elasticsearch namespace.

I think you're going to need to create a regular role in the tigera-elasticsearch namespace and bind kube controllers to that. Please give that a though and start a conversation with @doublek and I.

Reply:
yes, creating a roles in tigera-elasticsearch namespace and binding it to manager and kube-controller is definitely better option than clusterrole.